utils/fix_cmocka_xml.py

```diff
@@ -27,6 +27,9 @@
 # Two pass update on the files
 # Remove all testsuites tag out of the xml file.
 for file in files:
+    if os.path.isdir(file):
+		print("Skipping %s, it's a directory" % file)
```

Comment:
(pylint-syntax-error) inconsistent use of tabs and spaces in indentation (<unknown>, line 38)